_events/2021/2021-07-dei-speaker-series-ncwit.md

```diff
@@ -1,22 +1,33 @@
 ---
-title: The National Center for Women & Information Technology (NCWIT)
-subtitle: US-RSE DEI-WG Speaker Series
-expires: 2021-07-29
+title: POSTPONED
+subtitle: The National Center for Women & Information Technology (NCWIT) - US-RSE DEI-WG Speaker Series
+expires: 2021-08-31
```

Comment:
Changed this date, so hopefully it won't expire right away?

Reply:
Yes, this should be good. Sorry. I didn't look at the code yet when I wrote that, only the preview, which does still have the date. I just made a suggestion